extensions/2.0/Vendor/EXT_primitive_voxels/README.md

```diff
@@ -49,34 +61,195 @@ This extension allows primitives to use their attribute data to represent volume
+```json
+"accessors": [
+  {
+    "type": "SCALAR",
+    "componentType": 5122, // SHORT
+    "normalized": true
+  },
+  {
+    "type": "VEC3",
+    "componentType": 5126, // FLOAT
+  },
+  // ....
+]
+```
+
+...then it may define `noData` values for its corresponding attributes like so:
+
+```json
+"meshes": [
+  {
+    "primitives": [
+      {
+        "attributes": {
+          "_TEMPERATURE": 0,
+          "_DIRECTION": 1,
+          "_DATA_CONFIDENCE": 2
+        },
+        "mode": 2147483647,
+        "extensions": {
+          "EXT_primitive_voxels": {
+              "shape": 0,
+              "dimensions": [8, 8, 8],
+              "noData": {
+                "_TEMPERATURE": -32768,
+                "_DIRECTION": [-999.99, -999.99, -999.99]
+              }
+          }
+        }
+      }
+    ]
+  }
+]
+```
+
+Note that `_DATA_CONFIDENCE` intentionally does not specify a `noData` value. The attribute is expected to contain a valid value for every voxel cell.
+
+### Metadata
+
+This extension may be paired with the `EXT_structural_metadata` extension to convey more semantic information about the voxel attributes.
+
+```json
+{
+  "extensions": {
+    "EXT_structural_metadata": {
+      "schema": {
+        "classes": {
+          "voxels": {
+            "properties": {
+              "temperature": {
+                "type": "SCALAR",
+                "componentType": "UINT32",
+                "normalized": true,
+                "offset": 32.0,
+                "scale": 1.8
+              }
+            }
+          }
+        }
+      },
+      "propertyAttributes": [
+        {
+          "class": "voxels",
+          "properties": {
+            "temperature":{
+              "attribute": "_TEMPERATURE"
+            }
+          }
+        }
+      ]
+    }
+  },
+  "meshes": [
+    {
+      "primitives": [
+        {
+          "attributes": {
+            "_TEMPERATURE": 0
+          },
+          "extensions": {
+            "EXT_primitive_voxels": {
+              "dimensions": [8, 8, 8],
+              "padding": {
+                "before": [1, 1, 1],
+                "after": [1, 1, 1]
+              }
+            },
+            "EXT_structural_metadata": {
+              "propertyAttributes": [0]
+            }
+          }
+        }
+      ]
+    }
+  ]
+}
+```
 
-The padding data must be supplied with the rest of the voxel data - this means if `dimensions` is `[d1, d2, d3]`, `padding.before` is `[b1, b2, b3]`, and `padding.after` is `[a1, a2, a3]`, the attribute must supply `(d1 + a1 + b1)*(d2 + a2 + b2)*(d3 + a3 + b3)` elements.
+`EXT_structural_metadata` may also specify a `noData` value for a property attribute property. Runtimes have the liberty to decide what will happen if `EXT_primitive_voxels` also provides a different `noData` value for the same attribute.
```

Comment:
Maybe suggest that they _should_ match, just so it's clear that they represent the same thing.